tests/test_cases/stat.c

```diff
@@ -10,7 +10,7 @@
 
 extern char **environ;
 
-int (* const f)(const char *pathname, struct stat *statbuf) = lstat;
+int (* const f)(const char *pathname, struct stat *statbuf) = stat;
```

Comment:
I was hoping you would remove this "f" function that is defined as stat and just call stat directly in the main code.